src/api/parser/src/utils/__mocks__/supabase.js

```diff
@@ -0,0 +1,42 @@
+let feeds = [];
+let feedIds = new Set();
```

Comment:
When are you actually using this? I know that you use it in `__setMockFeeds`, but I don't think it has any members added, so it will always be empty, wouldn't it?

Reply:
I forgot to add the code that uses `feedIds`, I updated the way I use it, it doesn't look optimal now. Will try to make it work.